.circleci/smoketest_docker.sh

```diff
@@ -35,6 +35,9 @@ log_mode=$2
 # Max seconds before the test hard fails
 max_wait=$3
 
+# Name of a tested image
```

Comment:
Do we use circleci?

Reply:
We don't, but some of the tests for k8s and docker are still located here from the times where there were used in circleci.